packages/web-components/fast-components/src/radio-group/radio-group.ts

```diff
@@ -0,0 +1,217 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import {
+    keyCodeArrowDown,
+    keyCodeArrowLeft,
+    keyCodeArrowRight,
+    keyCodeArrowUp,
+    keyCodeEnter,
+} from "@microsoft/fast-web-utilities";
+import { RadioControl } from "../radio";
+
+export class RadioGroup extends FASTElement {
+    @attr({ attribute: "readonly", mode: "boolean" })
+    public readOnly: boolean;
+    private readOnlyChanged(): void {
+        const filteredRadios = this.getFilteredRadioButtons();
+        if (filteredRadios !== undefined) {
+            filteredRadios.forEach((radio: HTMLInputElement) => {
+                if (this.disabled) {
+                    radio.setAttribute("readonly", "");
+                } else {
+                    radio.removeAttribute("readonly");
+                }
+            });
+        }
+    }
+
+    @attr({ attribute: "disabled", mode: "boolean" })
+    public disabled: boolean;
+    private disabledChanged(): void {
+        const filteredRadios = this.getFilteredRadioButtons();
+        if (filteredRadios !== undefined) {
+            filteredRadios.forEach((radio: HTMLInputElement) => {
+                if (this.disabled) {
+                    radio.setAttribute("disabled", "");
+                } else {
+                    radio.removeAttribute("disabled");
+                }
+            });
+        }
+    }
+
+    @attr
+    public name: string;
+    protected nameChanged(): void {
+        this.getFilteredRadioButtons().forEach((radio: HTMLInputElement) => {
+            radio.setAttribute("name", this.name);
+        });
+    }
+
+    @attr
+    public value: string;
+
+    @observable slottedRadioButtons: RadioControl[];
+    private selectedRadio: RadioControl | null;
+
+    constructor() {
+        super();
+        this.addEventListener("keydown", this.keydownHandler);
+    }
+
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.addEventListener("change", this.handleRadioChange);
```

Comment:
I think you'll want to remove these listeners in the `disconnectedCallback`

Reply:
rob said not to do that, looking up again why

Reply:
I moved this to the constructor so below now applies...originally the way I had it yes, It should have been removed there
from Rob, 
the constructor is only called once but connect and disconnect can be called multiple times
you can add the listener in your constructor and not worry about removing it as long as you are only adding it to your own DOM
if you are listening to other elements outside of your own DOM, then you would need to add them in connect and remove them in disconnect